kihuha/iReporter-db
# #162485549 - Admin can change the status of an incident record

#### What does this PR do?
Contains code for implementing Admin incident status update

#### Description of the task to be completed
Have the following endpoint working
`<localhost>/api/v2/incidents/3/edit`
```request body
{
	"status": "accepted"
}
```
#### How should this be tested
- Clone the repository and navigate to the root folder. Start the server by typing `python run.py`.
- Create a user by sending a post request to `<localhost>/api/v2/users` using postman. The request body should match the format below
```
{
	"username": "kihuha",
	"email":"darius@gmail.com",
	"password": "darius",
	"firstname":"Darius",
	"lastname":"Njihia",
	"othernames":"Kihuha",
	"isadmin":"True",
	"phonenumber":"+254717309379"
}
```
- Send a post request to `<localhost>/api/v2/users/login` to generate an authentication token. Use this token in the header of the preceeding URL request
- Send a patch request to `<localhost>/api/v2/incidents/<int:id>/edit` with the id matching the id of the incident record you would like to edit. The body should conform to the format shown in the description
- The request will only be successful if the user logged in is an admin

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/162485549
